Reed/DefinitionView/viewmodels/DefinitionViewModel.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  DefinitionModalViewModel.swift
+//  Reed
+//
+//  Created by Hugo Zhan on 11/3/20.
+//  Copyright © 2020 Roger Luo. All rights reserved.
+//
+
+import SwiftUI
+
+class DefinitionViewModel: ObservableObject {
+    @Published var title = ""
+    @Published var currReading = ""
+    @Published var termsArr = [Terms]()
+    @Published var defsArr = [Defs]()
+    
+    let dictionaryFetcher = DictionaryFetcher()
+    
+    func generateContent(from tappedWord: String) {
+        title = tappedWord
+        termsArr.removeAll()
+        defsArr.removeAll()
+        let entries = dictionaryFetcher.fetchEntries(of: tappedWord)
+        if entries.count > 0 && entries[0].readings[0].terms.count > 0 && entries[0].readings[0].terms[0] == tappedWord {
```

Comment:
These should use `.endIndex`.
https://developer.apple.com/documentation/swift/array/1539310-endindex